src/Chainweb/Mempool/InMem.hs

```diff
@@ -456,7 +507,7 @@ getPendingInMem cfg nonce lock since callback = do
 
 ------------------------------------------------------------------------------
 clearInMem :: MVar (InMemoryMempoolData t) -> IO ()
-clearInMem lock = do
+clearInMem lock =
```

Comment:
I don't know why lint tools complain about these "superfluous" `do` statements. OK, you've cluttered up this PR by "cleaning it up", but now if someone else has to come in and extend this function the `do` needs to come back in.

To me, from a turd-polishing perspective the PR noise triggers my OCD as much or even more than the harmless lingering `do` does.